about.html

```diff
@@ -8,57 +8,75 @@
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
-
-</head>
-
+  <div>
+    <div class="logo">
+        <img src="img/lambda-black.png">
+        <nav>
+            <a href="#">Home</a>
+            <a href="#">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a> 
+            <a href="#">Contact</a>
+        </nav>
+        
+    </div>
+    <div class="img">
+        <img src="img/jumbo-about.png">
+    </div>
+    </div>
+    </head>
 <body>
     <div class="container about-page">
-     
+    <div class="firstParagragh">
     About
-
+    <br><br>
     Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
+</div>
+<div class="box1">
+<div class="strategy">
     <img src="img/about-plan.png" alt="strategy">
-    
+  <p><br>
     Strategy
-    
+    <br><br>
```

Comment:
There are many articles that debate using the <br> tags.  My first web site I had <br> all over the place, I didn't use <p>.  It was hard to read and it was not descriptive.  In place of <br> use css to space between elements, for example, if you wrapped Strategy in a <h3> tag.  you could style is by doing strategy h3 { margin: 10px; }.  This will put a margin around the Strategy and that gives spacing on the top, left, bottom, right.